roles/ceph-rgw/tasks/docker/rgw_pool_pgs_docker.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: create rgw pools if create_pools is defined
+  block:
+   # Get the short hostname of the first monitor to run the ceph command.
+   - name: find first monitor
+     set_fact:
+       ceph_first_monitor_hostname: "{{ hostvars[item].ansible_hostname }}"
+     with_items: "{{ groups['mons'][0] }}"
+
+   - name: set_fact docker_exec_cmd
+     set_fact:
+       docker_exec_cmd: "docker exec ceph-mon-{{ ceph_first_monitor_hostname }}"
+
+   # Execute the command on the first monitor, just once.
+   - name: create rgw pools
+     command: "{{ docker_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd pool create {{ item.key }} {{ item.value.pg_num }}"
+     changed_when: false
+     with_dict: "{{ create_pools }}"
+     delegate_to: "{{ groups['mons'][0] }}"
```

Comment:
do not hardcore `'mons'`, use the variable `mon_group_name`

Reply:
You are right. I guess sth like this would do the trick: {{ groups[mon_group_name][0] }}.

Reply:
yes

Reply:
Btw @guits, main playbooks have hardcoded group names in the host section:
* https://github.com/ceph/ceph-ansible/blob/master/site.yml.sample#L4
* https://github.com/ceph/ceph-ansible/blob/master/site-docker.yml.sample#L4

Maybe I'm missing sth, but In that case, shouldn't they be changed for sth like this??

  hosts:
    - "{{ mon_group_name|default('mons') }}"
    - "{{ osd_group_name|default('osds') }}"
    - "{{ mds_group_name|default('mdss') }}"
....

Reply:
@jtudelag no, it won't work i think because if i'm not mistaking, at this point `mon_group_name` is still an unknown variable even if defined in `group_vars/`